src/emitters/emitter.sqns.ts

```diff
@@ -716,6 +717,13 @@ export class SqnsEmitter implements IEmitter {
       });
       throw new Error(`Failed to parse message`);
     }
+
+    const payloadStructureVersion = message.messageAttributes?.PayloadVersion.StringValue;
```

Comment:
tested this locally, need to update parseDataFromMessage to parse messageAttributes as well.
for SQS it is in `receivedMessage.messageAttributes` 
for SNS it is in  `JSON.parse(receivedMessage.body).MessageAttributes`

Reply:
Also for SNS the structure seems to be `PayloadVersion.Value`
For SQS the structure seems to be `PayloadVersion.stringValue`

Reply:
resolved
please verify